g11n-ws/vip-manager-l10n-agent/src/main/java/com/vmware/l10agent/schedule/SourceRequestCron.java

```diff
@@ -182,7 +195,7 @@ public void lauchInstructToSync() {
 	public void syncToInternali18nManager() {
 	
 			Set<RecordModel> set = new HashSet<RecordModel>();
-			while (!TaskSysnQueues.SendComponentTasks.isEmpty()) {
+			while (!TaskSysnQueues.SendComponentTasks.isEmpty() && recoveredDir) {
```

Comment:
Is it possible that this method is executed before `@PostConstruct`?

Reply:
Yes